R/dalys.R

```diff
@@ -0,0 +1,169 @@
+##' Calculate dalys
+##'
+##' @title DALYs calculation
+##'
+##' @param con You can be \code{readonly} user to run this function.
+##' But if you need to import dalys for Ferrari, Li and LiST (201710gavi), you can use \code{import} user.
+##' @param dalys_src Dalys calcualtion parameters. This is currently a csv file in orderly_report i1400.
+##' We will surely move that file to model_documentation once it is complete.
+##' @param touchstone_name touchstone for which dalys are calcualted
+##' @param year_min minimal year
+##' @param year_max maximum year
+##' @param vimc_dalys_only set to be TRUE if we are only interested in Ferrari, Li and LiST (201710gavi)
+##'
+##' @export
+calculate_dalys <- function(con, dalys_src, touchstone_name, year_min, year_max, vimc_dalys_only = TRUE){
```

Comment:
Remove `dalys_src` from the argument list and instead do

```
daly_source <- read_csv(system.file("daly_parameters.csv", package = "jenner", mustWork = TRUE))
```

Reply:
There is an alternative way of doing this with sysdata.rda (see R-exts or Hadley's packages book) but this way will be more transparent if speed is not a problem

Reply:
do the `read_csv` only within the `create_dalys_parameters` function

Reply:
Done.